modules/govuk/manifests/apps/email_alert_api/checks.pp

```diff
@@ -37,20 +37,22 @@
   @@icinga::check::graphite { 'email-alert-api-warning-subscription-contents':
     ensure    => $ensure,
     host_name => $::fqdn,
-    target    => 'transformNull((keepLastValue(stats.gauges.govuk.email-alert-api.subscription_contents.warning_total))',
-    warning   => '1',
+    target    => 'transformNull(keepLastValue(stats.gauges.govuk.email-alert-api.subscription_contents.warning_total))',
+    warning   => '0',
+    critical  => '100000000', #set to a high value which won't be reached
```

Comment:
I think what a comment here could explain is the why. So why is this set to a high value that won't be reached?